configs/webpack-config-compass/src/loaders.ts

```diff
@@ -119,6 +119,21 @@ export const javascriptLoader = (args: ConfigArgs, web = false) => ({
   },
 });
 
+export const sourceMapLoader = (args: ConfigArgs) => ({
+  test: /\.(mjs|c?jsx?|tsx?)$/,
```

Comment:
![Untitled](https://github.com/mongodb-js/compass/assets/899444/cd64744e-e8e7-443f-b552-3fa04735d36e)

Reply:
Hahahaha, saving this 🤣